src/Command/Setup.php

```diff
@@ -55,11 +55,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $io = new SymfonyStyle($input, $output);
         $io->setDecorated(true);
-        $io->comment(sprintf('Composing your <info>%s</info> environment', (string) $envType));
+        $io->comment(sprintf('Start composing your <info>%s</info> environment', (string) $envType));
 
-        $vars = new AppVendor(
-            $io->ask('Vendor name', null, [AppVendor::class, 'validate']),
-            $io->ask('App name', null, [AppVendor::class, 'validate'])
+        $vars = new Host(
+            $io->ask('App name', 'app', [Host::class, 'validate']),
+            $io->ask('Vendor name (optional)', false, [Host::class, 'validate'])
```

Comment:
Using `false` as default value is a workaround. See https://github.com/symfony/symfony/pull/20141